client/src/pages/TopicPage.jsx

```diff
@@ -1,31 +1,31 @@
-import { useState, useEffect, useContext } from 'react';
-import axios from 'axios';
+import { useState, useContext } from 'react';
 import TopicDetails from '../components/Topic/TopicDetails';
 import EditTopic from '../components/Topic/EditTopic';
 import CreateATopic from '../components/Topic/CreateATopic';
 import { AuthContext } from '../context/auth.context';
+import { useFetch } from '../hooks/useFetch';
+import Loading from '../components/Loading/Loading';
 
 export default function TopicPage() {
-  const [topics, setTopics] = useState([]);
   const [selectedTopicId, setSelectedTopicId] = useState('all');
   const [selectedTopic, setSelectedTopic] = useState(null);
   const [refreshTopics, setRefreshTopics] = useState(false);
   const [addTopic, setAddTopic] = useState(false);
   const { isLoggedIn, user } = useContext(AuthContext);
 
-  useEffect(() => {
-    axios
-      .get('/api/topic/list-topics')
-      .then((response) => {
-        setTopics(response.data.topics);
-        setRefreshTopics();
```

Comment:
this use effect is usefull also for setRefreshTopic, how do you set it with the new implementation?

Reply:
Now is fixed, please check it out! 🙏🏼